library/src/main/scala/com/monsanto/arch/kamon/prometheus/PrometheusExtension.scala

```diff
@@ -43,6 +50,13 @@ class PrometheusExtension(system: ExtendedActorSystem) extends Kamon.Extension {
   /** The Spray endpoint. */
   val route: Route = endpoint.route
 
+  /** For internal use (healthchecks, other routes, etc.) */
+  def requestSnapshot() = Option(snapshot.get())
+
+  def flushBuffer() = {
+    buffer ! FlushBuffer
+  }
+
```

Comment:
With the updates I am making to the patch, `flushBuffer` is no longer needed.  For the moment, I am going to remove `requestSnapshot` because I think I would like to go ahead and break up the library into multiple parts:

* the core library that will collect the snapshots and expose them (similar to `requestSnapshot`)
* separate libraries for endpoints for different web libraries, e.g. Spray, Akka, Play

I will make it a priority to go ahead and try to get that done this week and cut a new release.  This way, for your own application, you can use the core library without bringing in Spray.  Does that sound reasonable to you?